voxpupuli-puppet-lint-plugins.gemspec

```diff
@@ -11,15 +11,14 @@ Gem::Specification.new do |s|
   s.licenses    = 'AGPL-3.0'
 
   # some of the plugins claim to require Ruby 2.4.0 or newer
```

Comment:
I'd say that we set it to 2.5 since that's compatible with Puppet 6 AIO, while also being able to reasonably support Ruby 3.

Reply:
fixed